packages/in_app_purchase/in_app_purchase_storekit/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
```

Comment:
Is this supposed to be a version number? What is `NEXT`?

Reply:
I used the `update-release-info` tool with the `--version=minimal` option.
This generated the `NEXT` automatically, indicating that the change does not required a version update.

But I guess from reading [the docs](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog), there shouldn't be an entry to the CHANGELOG at all, since this PR only affects development of the package, i.e. is a test-only change.
> All version changes must have an accompanying CHANGELOG update. Even version-exempt changes should usually update CHANGELOG by adding a special NEXT entry at the top of CHANGELOG.md (unless they only affect development of the package, such as test-only changes):

How would you interpret this? Should I remove the CHANGELOG entry again, leave it as is, or add a proper version change?

Reply:
This is a tool bug; it wasn't updated when the repo policy changed (filed https://github.com/flutter/flutter/issues/113941). You can just just remove the entry entirely, yes.